src/components/Footer/Footer.stories.mdx

```diff
@@ -28,25 +28,128 @@ import DSProvider from "../../theme/provider";
 
 | Component Version | DS Version |
 | ----------------- | ---------- |
-| Added             | `1.1.0`    |
-| Latest            | `1.1.0`    |
+| Added             | `TBD`      |
+| Latest            | `TBD`      |
 
 ## Table of Contents
 
 - [Overview](#overview)
 - [Component Props](#component-props)
 - [Accessibility](#accessibility)
+- [With DS Template Components](#with-ds-templates)
 
 ## Overview
 
 <Description of={Footer} />
 
+This component is expected to be rendered at the end of a webpage, before the
+closing `</body>` tag. There are no expected props for this component so it can
+be rendered simply as `<Footer />`.
+
+```tsx
+<body>
+  <header>...</header>
+  <main>...</main>
+
+  <!-- Add the Footer component here -->
+  <Footer />
+</body>
+```
+
 ## Component Props
 
 <Canvas withToolbar>
-  <Story name="Footer">{(args) => <Footer />}</Story>
+  <Story name="Footer">
+    <Footer />
+  </Story>
 </Canvas>
 
-<ArgsTable story="Footer" />
+Note: There are custom props for the `Footer` component.
 
 ## Accessibility
+
+The `Footer` component renders as an HTML `<footer>` element with a WAI-ARIA role
+of `role="contentinfo"`. This is the common practice for the footer landmark
+region on a page to maximize its accessibility in browsers and assistive
+technologies.
+
+Rendering this `Footer` component, or similar HTML landmark region components,
```

Comment:
It would be nice to add more weight to this statement.  Maybe add somethign like this to the beginning:

_Special consideration should be made regarding the placement of this component in the DOM because..._